### PR TITLE
Vertical graphs are now working

### LINKED ISSUE #167: vertical view of graphs stopped working
Not sure when this happened, but it was pretty recently.

### PR DESCRIPTION
**Pull request by diogoscf**
Fixes #167 
Hard to find the cause, but easy to fix!
The vertical graphs are somewhat ugly, but they already were before they got broken.

### CODE REVIEW

**Comment by ftyers** [comment]
@diogoscf can you add a screenshot ?

**Comment by diogoscf** [comment]
How it looks now:
new_vertical
For comparison, how it looked in the last commit before it broke:
old_vertical

File: standalone/lib/visualiser.js [hunk 1]
```diff
@@ -697,7 +697,7 @@ function cleanEdges() {
  */
 function setEdgePosition(thisEdge, thisHeight, coef, diff) {
 	if (!LEFT_TO_RIGHT) {coef *= -1}; // support for RTL
-	if (VERT_ALIGNMENT) {edgeDep.ctrl = [45, 45, 45, 45]};
+	//if (VERT_ALIGNMENT) {edgeDep.ctrl = [45, 45, 45, 45]};
```
Comment (gstark0) [inline standalone/lib/visualiser.js]: It's the same as before, why is it commented?

**Comment by jonorthwash** [comment]
Could you show a screenshot of connected nodes too?

**Comment by jonorthwash** [comment]
Also, which commit broke it?

**Comment by diogoscf** [comment]
The commit that broke it was 596379f

**Comment by diogoscf** [comment]
It's a bit broken:

new_vertical_withnodes

But it wasn't before... Hadn't noticed that...

old-vertical_withnodes

**Comment by jonorthwash** [comment]
Oh, I know why it's doing that.  It's staggering the arcs so that incoming and outgoing arcs don't overlap, which is by design.  Could you test it out with some real trees from a treebank (and share screenshots)?

**Comment by maryszmary** [comment]
It doesn't work for (e.g.) this tree:
> 1	N'	ne	ADV	adv	Polarity=Neg	4	advmod	_	_
2	int	bezañ	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux	_	_
3	ket	ket	ADV	adv	_	4	advmod	_	_
4	aet	mont	VERB	vblex	Tense=Past|VerbForm=Part	0	root	_	_
5	war-raok	war-raok	ADV	adv	_	4	advmod	_	_
6	.	.	PUNCT	sent	_	4	punct	_	_

giving the following error messages from cytoscape:

> The style property `text-margin-x: ` is invalid

> cytoscape.min.js:1 Do not assign mappings to elements without corresponding data (e.g. ele `ep01` for property `text-margin-x` with data field `length`); try a `[length]` selector to limit scope to elements with `length` defined

> cytoscape.min.js:10 Uncaught TypeError: Cannot set property 'mapping' of null

(horizontal alignment does work):
image

**Comment by diogoscf** [comment]
Huh. Those are the errors it gave before I fixed it and thay stopped after I fixed the code. Are you sure you aren’t using the code from before the fix @maryszmary ?

**Comment by diogoscf** [comment]
And a two more examples:
somenode
othernode

**Comment by diogoscf** [comment]
They now look good again!
Examples:
othernodes
somenodes
extranodes

**Comment by jonorthwash** [comment]
The arrows with the relation tags are pretty meaningless like this...

**Comment by maryszmary** [comment]
@diogoscf , my mistake: it works. Thank you!

**Pull request merged**